src/sdk/modules/passkeyValidator/utils.ts

```diff
@@ -0,0 +1,152 @@
+import { p256 } from "@noble/curves/p256"
```

Comment:
This isn't in our dependencies either

Reply:
This library comes from other dependencies we use, it doesn't have to be in our package.json. It comes from "nexus", and "viem".

Reply:
nexus isn't in our list of dependencies either. If it's in viem then OK 👍

Reply:
But I thought viem had no dependencies too?

Reply:
You're right https://github.com/wevm/viem/blob/main/src/package.json#L142